lib/Controller/OutboxController.php

```diff
@@ -156,8 +155,8 @@ public function create(
 	 * @return JsonResponse
 	 */
 	#[TrapError]
-	public function createFromDraft(DraftsService $draftsService, int $id, int $sendAt = null): JsonResponse {
-		$draftMessage = $draftsService->getMessage($id, $this->userId);
+	public function createFromDraft(int $id, int $sendAt = null): JsonResponse {
+		$draftMessage = $this->service->getMessage($id, $this->userId);
```

Comment:
on updateDraft() the message has .sentAt value so it gets a  TYPE_OUTGOING that's why we're using the outboxService here

Reply:
So the new endpoint should be removed right? I can't see it used anywhere else

Reply:
Or we to be more in line with https://github.com/nextcloud/mail/issues/8631 update draft shouldn't change message type even if it has a sentAt

Reply:
> on updateDraft() the message has .sentAt value so it gets a TYPE_OUTGOING

is what we should stop doing. The conversion has to happen explicitly, not on the basis of sendAt being set or not